lib/travis/api/v3/model_renderer.rb

```diff
@@ -42,16 +42,17 @@ def self.available_attributes
       @available_attributes ||= superclass.available_attributes.dup
     end
 
-    def self.render(model, representation = :standard, **options)
+    def self.render(model, representation = nil, **options)
```

Comment:
Is changing the default value a good idea?

Reply:
there's now a default on the instance method `representation` so that individual renderers can overwrite that